### PR TITLE
Changed bors commit message and added a reference in the PR template

### PR DESCRIPTION
**Pull request by fhennig**
The description of the PR becomes the commit message of the merge commit after it is merged (see [here](https://github.com/stackabletech/druid-operator/commit/d74e1cdc1337bf57efe9aed4568dbb1fad2bdd6d) for an example).

This PR uses the `cut_body_after` property to cut off our review checklist. It also adds a short note to use bors for the merge.

~It also configures bors to do a squash merge.~ (Doesn't work)

### CODE REVIEW

File: template/bors.toml [hunk 1]
```diff
@@ -6,5 +6,7 @@ status = [
     'Run cargo deny (bans licenses sources)'
 ]
 delete_merged_branches = true
+use_squash_merge = true
 pr_status = [ 'license/cla' ]
-timeout_sec = 7200
\ No newline at end of file
+timeout_sec = 7200
+cut_body_after = "<!-- Commit message above. Everything below is not added to the message. Do not change this line! -->"
```
Comment (teozkr) [inline template/bors.toml:12]: :+1:

File: template/bors.toml [hunk 2]
```diff
@@ -6,5 +6,7 @@ status = [
     'Run cargo deny (bans licenses sources)'
 ]
 delete_merged_branches = true
+use_squash_merge = true
```
Comment (teozkr) [inline template/bors.toml:9]: We had issues where squash merges didn't close the PR properly.

I'm also curious about what use-case you're trying to solve with squashing? Git itself can show the history as if every merge was a squash with `git log --first-parent`.

Comment (fhennig) [reply template/bors.toml:9]: That was mostly to emulate how we did it before I thought maybe it was on accident that you missed that. I also kind of liked that all my "WIP" commits don't show up in the final thing but I'm happy either way

Comment (fhennig) [reply template/bors.toml]: I've removed it for now

**Comment by maltesander** [comment]
Included here https://github.com/stackabletech/operator-templating/pull/46

**Pull request closed without merging**